tests/apply-shim.html

```diff
@@ -273,6 +290,20 @@
         }
       });
     });
+    suite('!important', function() {
+      var ast; // eslint-disable-line no-unused-vars
+      suiteSetup(function() {
+        var info = prep('important');
+        ast = info.ast;
+      });
+
+      test('!important is preserved and overrides other properties', function() {
+        var application = ast.rules[1];
+        assert.match(application.cssText, /^border:\s*var\(--mixin-important_-_border\)\s*!important/);
+        assert.notMatch(application.cssText, /background-color:\s*var\(--mixin-important_-_background-color\)\s*!important/);
+        assert.match(application.cssText, /border:\s*var\(--mixin_-_border,\s*var\(--mixin-important_-_border\)\)/);
```

Comment:
Maybe split this cases up so that debugging of the test cases becomes easier. For example:

1. `!important in earlier selector correctly translates to !important in mixin properties`
1. `Properties without !important in a mixin with !important are treated normally`
1. `two mixins with both !important are treated as regular ordered selectors`
1. `fallback of unrelated property are untouched`